tasks/Restaurant.tex

```diff
@@ -0,0 +1,88 @@
+	\begin{itemize}[nosep]
+		\item \textbf{Barman:} A \textit{Professional Barman} (member of the TC) will be at the other side of the \textit{Kitchen-bar} to take the order provided by the robot and serve it in the official tray.
+		\item \textbf{Items:} Items will be located on the Kitchen-bar next to the \textit{Professional Barman}.
+	\end{itemize}
+\end{itemize}
+
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Safety!} This test takes place in a public area. That is, there may be people standing, sitting or walking around the area throughout the test. The robot is expected to not even slightly touch anything and is immediately stopped in case of danger.
+
+	\item \textbf{Referees and guidance:} For safety reasons, the referees in this test are TC members. One of the referees follows the robot and is always in reach of the emergency button.
+
+    \item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied per order as follows.
+        \begin{itemize}[nosep]
+          \item Needing guidance during the deliver phase causes a reduction of 40\% in scoring.
+          \item Needing assistance for picking up an item/the tray causes a reduction of 10\% in scoring.
+          \item Have a human place the item in a basket causes a reduction of 20\% in scoring.
+          \item Have the guest take the item out of the basket causes a reduction of 20\% in scoring.
+          \item Asking the barman at which side the \textit{Kitchen-bar} is causes a reduction of 5\% in the total scoring.
+        \end{itemize}
+
+	\item \textbf{Start:} There is no fixed start signal in this test, it starts when both robots are ready (though within a reasonable time).
+
+	\item \textbf{Order:} The way the user provides information to the robot is up to the robot's team. A natural interaction is preferred.
+
+	\item \textbf{Location:} This test can be arranged in any real restaurant or shopping mall. If this is not possible, the test can be conducted in an arbitrary room containing the appropriate locations. The only requirement is that this room is not part of the arena and that the teams do not know the room beforehand.
+	  The exact location, including the object and delivery locations, will be defined by the technical committee on site (and in corporation with the local organization). In addition, to avoid unnecessary time investment for navigation, the distances between tables and the \enquote{Kitchen Bar} will be minimal. Because the location can be anywhere, nearby electrical outlets and Wi-Fi access can't be guaranteed.
+
+	\item \textbf{Disturbances from outside:} If a person from the audience (severely) interferes with the robot in a way that makes it impossible to solve the task, the teams may repeat the test immediately.
```

Comment:
Ask the barman?

Reply:
Difficult when both robots are patrolling the restaurant. Should they go back and ask? That would again seem pretty weird and unnatural to audiences. Which waiter goes back to the bar and asks if they are allowed to take an order.

Reply:
I think the patrolling idea was discarded in the past meeting. Basically the robot ask the barman what to do and if it can, the barman takes over the test (guide robot to table, take order for robot, guide robot back, dispatch order, deliver order). The more work the barman does, the higher the chances your robot scores zero and gets fired.

Reply:
Yep pratrolling is discarded.